ICE: `Stored empty attributes for 3`

<!--
ICE: Rustc ./a.rs '' 'thread 'rustc' panicked at compiler/rustc_ast_lowering/src/lib.rs:618:17: 'Stored empty attributes for 11'', 'thread 'rustc' panicked at compiler/rustc_ast_lowering/src/lib.rs:618:17: 'Stored empty attributes for 11''
File: /home/gh-matthiaskrgr/vcs/github/rust_misc_stuff/tests/ICE/a.rs
-->

auto-reduced (treereduce-rust):
````rust
fn main() {
    let y = #[repr(uwu(4))]
    (&id(5)); //~ ERROR
}

````

original:
````rust
//! This test does not try to check if the output artifacts are valid.

pub fn opt_str2<'a>(maybestr: &'a Option<String>) -> &'static str {
    match *maybestr {
        None => "(none)",
        Some(ref s) => {
            let s: &'a str = s;
            s
            //~^ ERROR lifetime may not live long enough
        }
    }
}

fn main() {
    let x = Some(127.99, 127)
    let y = #[repr(uwu(4))](&id(5));  //~ ERROR
    let _ = &in_scope2;
}

````

Version information
````
rustc 1.88.0-dev
binary: rustc
commit-hash: unknown
commit-date: unknown
host: x86_64-unknown-linux-gnu
release: 1.88.0-dev
LLVM version: 20.1.2

````

Possibly related line of code:
https://github.com/rust-lang/rust/blob/a4166dabaae56ab0d35a7825c3e7008778eacf34/compiler/rustc_ast_lowering/src/lib.rs#L612-L624

Command:
`/home/gh-matthiaskrgr/.rustup/toolchains/local-debug-assertions/bin/rustc `

<details><summary><strong>Program output</strong></summary>
<p>

```
error[E0658]: attributes on expressions are experimental
 --> /tmp/icemaker_global_tempdir.uaDhzO8JHg0u/rustc_testrunner_tmpdir_reporting.QmGyLLTXjxH5/mvce.rs:2:13
  |
2 |     let y = #[repr(uwu(4))]
  |             ^^^^^^^^^^^^^^^
  |
  = note: see issue #15701 <https://github.com/rust-lang/rust/issues/15701> for more information
  = help: add `#![feature(stmt_expr_attributes)]` to the crate attributes to enable

error[E0425]: cannot find function `id` in this scope
 --> /tmp/icemaker_global_tempdir.uaDhzO8JHg0u/rustc_testrunner_tmpdir_reporting.QmGyLLTXjxH5/mvce.rs:3:7
  |
3 |     (&id(5)); //~ ERROR
  |       ^^ not found in this scope
  |
help: consider importing this function
  |
1 + use std::process::id;
  |

error[E0552]: unrecognized representation hint
 --> /tmp/icemaker_global_tempdir.uaDhzO8JHg0u/rustc_testrunner_tmpdir_reporting.QmGyLLTXjxH5/mvce.rs:2:20
  |
2 |     let y = #[repr(uwu(4))]
  |                    ^^^^^^
  |
  = help: valid reprs are `Rust` (default), `C`, `align`, `packed`, `transparent`, `simd`, `i8`, `u8`, `i16`, `u16`, `i32`, `u32`, `i64`, `u64`, `i128`, `u128`, `isize`, `usize`


thread 'rustc' panicked at compiler/rustc_ast_lowering/src/lib.rs:618:17:
Stored empty attributes for 3
stack backtrace:
   0:     0x74853166af33 - std::backtrace_rs::backtrace::libunwind::trace::h058677524833a6b8
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/../../backtrace/src/backtrace/libunwind.rs:117:9
   1:     0x74853166af33 - std::backtrace_rs::backtrace::trace_unsynchronized::h98e3bf1bf092de3c
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/../../backtrace/src/backtrace/mod.rs:66:14
   2:     0x74853166af33 - std::sys::backtrace::_print_fmt::h0693e8ebeb886cfa
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/sys/backtrace.rs:66:9
   3:     0x74853166af33 - <std::sys::backtrace::BacktraceLock::print::DisplayBacktrace as core::fmt::Display>::fmt::hffadf8d683d43b08
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/sys/backtrace.rs:39:26
   4:     0x74852c6b34a1 - core::fmt::rt::Argument::fmt::h51b5148115b1cc73
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/core/src/fmt/rt.rs:178:76
   5:     0x74852c6b34a1 - core::fmt::write::h96b1945cb8d5edf6
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/core/src/fmt/mod.rs:1480:21
   6:     0x7485316260e9 - std::io::default_write_fmt::hf45f5329379f552b
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/io/mod.rs:639:11
   7:     0x7485316260e9 - std::io::Write::write_fmt::h739b0d712cccf231
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/io/mod.rs:1914:13
   8:     0x74853166add3 - std::sys::backtrace::BacktraceLock::print::h909fc60c9b33bda4
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/sys/backtrace.rs:42:9
   9:     0x748531657199 - std::panicking::default_hook::{{closure}}::h65aaf55d6ed811e6
  10:     0x748531656fd3 - std::panicking::default_hook::hf2fdacddcf45ad0a
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:327:9
  11:     0x74852db8f4cb - rustc_driver_impl[b803c1eededc3934]::install_ice_hook::{closure#1}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_driver_impl/src/lib.rs:1371:17
  12:     0x74852db8f4cb - <alloc[6dd86dc546ece569]::boxed::Box<rustc_driver_impl[b803c1eededc3934]::install_ice_hook::{closure#1}> as core[8d9672d5584755be]::ops::function::Fn<(&dyn for<'a, 'b> core[8d9672d5584755be]::ops::function::Fn<(&'a std[cd54381a177933ca]::panic::PanicHookInfo<'b>,), Output = ()> + core[8d9672d5584755be]::marker::Send + core[8d9672d5584755be]::marker::Sync, &std[cd54381a177933ca]::panic::PanicHookInfo)>>::call
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/alloc/src/boxed.rs:1980:9
  13:     0x748531657873 - <alloc::boxed::Box<F,A> as core::ops::function::Fn<Args>>::call::h11a4f2aaad42d4de
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/alloc/src/boxed.rs:1980:9
  14:     0x748531657873 - std::panicking::rust_panic_with_hook::h9df10e027007a66e
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:841:13
  15:     0x74853166b40c - std::panicking::begin_panic_handler::{{closure}}::h454b9a171b778b0f
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:706:13
  16:     0x74853166b209 - std::sys::backtrace::__rust_end_short_backtrace::heee200d47ed01be9
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/sys/backtrace.rs:168:18
  17:     0x74853165728f - __rustc[ed85a5c164681b92]::rust_begin_unwind
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:697:5
  18:     0x74852c6c25c0 - core::panicking::panic_fmt::hb36d374cbc631117
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/core/src/panicking.rs:75:14
  19:     0x74852cb391e9 - <rustc_ast_lowering[db0c7010d65b16f6]::LoweringContext>::make_owner_info
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_ast_lowering/src/lib.rs:618:17
  20:     0x74852cb42202 - <rustc_ast_lowering[db0c7010d65b16f6]::LoweringContext>::with_hir_id_owner::<<rustc_ast_lowering[db0c7010d65b16f6]::item::ItemLowerer>::with_lctx<<rustc_ast_lowering[db0c7010d65b16f6]::item::ItemLowerer>::lower_item::{closure#0}>::{closure#0}>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_ast_lowering/src/lib.rs:587:20
  21:     0x74852cbf98b0 - <rustc_ast_lowering[db0c7010d65b16f6]::item::ItemLowerer>::with_lctx::<<rustc_ast_lowering[db0c7010d65b16f6]::item::ItemLowerer>::lower_item::{closure#0}>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_ast_lowering/src/item.rs:61:9
  22:     0x74852cbfa99a - <rustc_ast_lowering[db0c7010d65b16f6]::item::ItemLowerer>::lower_item
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_ast_lowering/src/item.rs:102:9
  23:     0x74852cbfa99a - <rustc_ast_lowering[db0c7010d65b16f6]::item::ItemLowerer>::lower_node
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_ast_lowering/src/item.rs:80:41
  24:     0x74852cb37c32 - rustc_ast_lowering[db0c7010d65b16f6]::lower_to_hir
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_ast_lowering/src/lib.rs:449:9
  25:     0x74853032f3e5 - rustc_query_impl[2ae6f340e9b30694]::query_impl::hir_crate::dynamic_query::{closure#2}::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:288:9
  26:     0x74853032f3e5 - rustc_query_impl[2ae6f340e9b30694]::plumbing::__rust_begin_short_backtrace::<rustc_query_impl[2ae6f340e9b30694]::query_impl::hir_crate::dynamic_query::{closure#2}::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:572:18
  27:     0x7485302567f7 - rustc_query_impl[2ae6f340e9b30694]::query_impl::hir_crate::dynamic_query::{closure#2}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:646:25
  28:     0x7485302567f7 - <rustc_query_impl[2ae6f340e9b30694]::query_impl::hir_crate::dynamic_query::{closure#2} as core[8d9672d5584755be]::ops::function::FnOnce<(rustc_middle[e400e38c50169c83]::ty::context::TyCtxt, ())>>::call_once
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/core/src/ops/function.rs:250:5
  29:     0x74853012e162 - <rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false> as rustc_query_system[26c1762410ebf569]::query::config::QueryConfig<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>>::compute
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/lib.rs:124:9
  30:     0x74853012e162 - rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr::<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:493:66
  31:     0x74853012e162 - rustc_middle[e400e38c50169c83]::ty::context::tls::enter_context::<rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:60:9
  32:     0x74853012e162 - <std[cd54381a177933ca]::thread::local::LocalKey<core[8d9672d5584755be]::cell::Cell<*const ()>>>::try_with::<rustc_middle[e400e38c50169c83]::ty::context::tls::enter_context<rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/thread/local.rs:311:12
  33:     0x74853012e162 - <std[cd54381a177933ca]::thread::local::LocalKey<core[8d9672d5584755be]::cell::Cell<*const ()>>>::with::<rustc_middle[e400e38c50169c83]::ty::context::tls::enter_context<rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/thread/local.rs:275:15
  34:     0x74853012e162 - rustc_middle[e400e38c50169c83]::ty::context::tls::enter_context::<rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:57:5
  35:     0x74853012e162 - <rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query::<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:155:13
  36:     0x74853012e162 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context::<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:112:9
  37:     0x74853012e162 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_context::<rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:90:36
  38:     0x74853012e162 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_context_opt::<rustc_middle[e400e38c50169c83]::ty::context::tls::with_context<rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:79:18
  39:     0x74853012e162 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_context::<rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:90:5
  40:     0x74853012e162 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context::<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:103:5
  41:     0x74853012e162 - <rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query::<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:141:9
  42:     0x74853012e162 - rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr::<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:493:18
  43:     0x74853012e162 - rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job::<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt, false>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:429:9
  44:     0x74853012e162 - rustc_query_system[26c1762410ebf569]::query::plumbing::try_execute_query::<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt, false>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:372:13
  45:     0x7485305dc7a5 - rustc_query_system[26c1762410ebf569]::query::plumbing::get_query_non_incr::<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:814:32
  46:     0x7485305dc7a5 - stacker[1fe10eee22c7cf45]::maybe_grow::<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::get_query_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>
                               at /home/gh-matthiaskrgr/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/stacker-0.1.20/src/lib.rs:57:9
  47:     0x7485305dc7a5 - rustc_data_structures[48bbbe4422bc1485]::stack::ensure_sufficient_stack::<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::get_query_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_data_structures/src/stack.rs:21:5
  48:     0x7485305dc7a5 - rustc_query_system[26c1762410ebf569]::query::plumbing::get_query_non_incr::<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:814:5
  49:     0x7485305dc7a5 - rustc_query_impl[2ae6f340e9b30694]::query_impl::hir_crate::get_query_non_incr::__rust_end_short_backtrace
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:622:26
  50:     0x74852ec91c51 - rustc_middle[e400e38c50169c83]::query::plumbing::query_get_at::<rustc_query_system[26c1762410ebf569]::query::caches::SingleCache<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/query/plumbing.rs:182:17
  51:     0x74852ec91c51 - <rustc_middle[e400e38c50169c83]::query::plumbing::TyCtxtAt>::hir_crate
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/query/plumbing.rs:472:31
  52:     0x74852ec91c51 - <rustc_middle[e400e38c50169c83]::ty::context::TyCtxt>::hir_crate
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/query/plumbing.rs:463:35
  53:     0x74852ec91c51 - rustc_middle[e400e38c50169c83]::hir::provide::{closure#1}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/hir/mod.rs:195:23
  54:     0x74852ec91c51 - <rustc_middle[e400e38c50169c83]::hir::provide::{closure#1} as core[8d9672d5584755be]::ops::function::FnOnce<(rustc_middle[e400e38c50169c83]::ty::context::TyCtxt, rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId)>>::call_once
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/core/src/ops/function.rs:250:5
  55:     0x74853031c796 - rustc_query_impl[2ae6f340e9b30694]::query_impl::opt_hir_owner_nodes::dynamic_query::{closure#2}::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:288:9
  56:     0x74853031c796 - rustc_query_impl[2ae6f340e9b30694]::plumbing::__rust_begin_short_backtrace::<rustc_query_impl[2ae6f340e9b30694]::query_impl::opt_hir_owner_nodes::dynamic_query::{closure#2}::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:572:18
  57:     0x74853022a70b - rustc_query_impl[2ae6f340e9b30694]::query_impl::opt_hir_owner_nodes::dynamic_query::{closure#2}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:646:25
  58:     0x74853022a70b - <rustc_query_impl[2ae6f340e9b30694]::query_impl::opt_hir_owner_nodes::dynamic_query::{closure#2} as core[8d9672d5584755be]::ops::function::FnOnce<(rustc_middle[e400e38c50169c83]::ty::context::TyCtxt, rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId)>>::call_once
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/core/src/ops/function.rs:250:5
  59:     0x7485301a1cb1 - <rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false> as rustc_query_system[26c1762410ebf569]::query::config::QueryConfig<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>>::compute
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/lib.rs:124:9
  60:     0x7485301a1cb1 - rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr::<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:493:66
  61:     0x7485301a1cb1 - rustc_middle[e400e38c50169c83]::ty::context::tls::enter_context::<rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:60:9
  62:     0x7485301a1cb1 - <std[cd54381a177933ca]::thread::local::LocalKey<core[8d9672d5584755be]::cell::Cell<*const ()>>>::try_with::<rustc_middle[e400e38c50169c83]::ty::context::tls::enter_context<rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/thread/local.rs:311:12
  63:     0x7485301a1cb1 - <std[cd54381a177933ca]::thread::local::LocalKey<core[8d9672d5584755be]::cell::Cell<*const ()>>>::with::<rustc_middle[e400e38c50169c83]::ty::context::tls::enter_context<rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/thread/local.rs:275:15
  64:     0x7485301a1cb1 - rustc_middle[e400e38c50169c83]::ty::context::tls::enter_context::<rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:57:5
  65:     0x7485301a1cb1 - <rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query::<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:155:13
  66:     0x7485301a1cb1 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context::<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:112:9
  67:     0x7485301a1cb1 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_context::<rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:90:36
  68:     0x7485301a1cb1 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_context_opt::<rustc_middle[e400e38c50169c83]::ty::context::tls::with_context<rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:79:18
  69:     0x7485301a1cb1 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_context::<rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:90:5
  70:     0x7485301a1cb1 - rustc_middle[e400e38c50169c83]::ty::context::tls::with_related_context::<<rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>::{closure#0}, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context/tls.rs:103:5
  71:     0x7485301a1cb1 - <rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt as rustc_query_system[26c1762410ebf569]::query::QueryContext>::start_query::<rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>::{closure#0}>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:141:9
  72:     0x7485301a1cb1 - rustc_query_system[26c1762410ebf569]::query::plumbing::execute_job_non_incr::<rustc_query_impl[2ae6f340e9b30694]::DynamicConfig<rustc_data_structures[48bbbe4422bc1485]::vec_cache::VecCache<rustc_span[d487cbcc0f7848f3]::def_id::LocalDefId, rustc_middle[e400e38c50169c83]::query::erase::Erased<[u8; 8usize]>, rustc_query_system[26c1762410ebf569]::dep_graph::graph::DepNodeIndex>, false, false, false>, rustc_query_impl[2ae6f340e9b30694]::plumbing::QueryCtxt>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:493:18


<snip>

 176:     0x74852dbf8758 - std[cd54381a177933ca]::panicking::try::do_call::<core[8d9672d5584755be]::panic::unwind_safe::AssertUnwindSafe<<std[cd54381a177933ca]::thread::Builder>::spawn_unchecked_<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_with_globals<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_pool_with_globals<rustc_interface[bb7f2a6353b38934]::interface::run_compiler<(), rustc_driver_impl[b803c1eededc3934]::run_compiler::{closure#0}>::{closure#1}, ()>::{closure#0}, ()>::{closure#0}::{closure#0}, ()>::{closure#1}::{closure#0}>, ()>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:589:40
 177:     0x74852dbf8758 - std[cd54381a177933ca]::panicking::try::<(), core[8d9672d5584755be]::panic::unwind_safe::AssertUnwindSafe<<std[cd54381a177933ca]::thread::Builder>::spawn_unchecked_<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_with_globals<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_pool_with_globals<rustc_interface[bb7f2a6353b38934]::interface::run_compiler<(), rustc_driver_impl[b803c1eededc3934]::run_compiler::{closure#0}>::{closure#1}, ()>::{closure#0}, ()>::{closure#0}::{closure#0}, ()>::{closure#1}::{closure#0}>>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:552:19
 178:     0x74852dbf8758 - std[cd54381a177933ca]::panic::catch_unwind::<core[8d9672d5584755be]::panic::unwind_safe::AssertUnwindSafe<<std[cd54381a177933ca]::thread::Builder>::spawn_unchecked_<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_with_globals<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_pool_with_globals<rustc_interface[bb7f2a6353b38934]::interface::run_compiler<(), rustc_driver_impl[b803c1eededc3934]::run_compiler::{closure#0}>::{closure#1}, ()>::{closure#0}, ()>::{closure#0}::{closure#0}, ()>::{closure#1}::{closure#0}>, ()>
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/panic.rs:359:14
 179:     0x74852dbf8758 - <std[cd54381a177933ca]::thread::Builder>::spawn_unchecked_::<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_with_globals<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_pool_with_globals<rustc_interface[bb7f2a6353b38934]::interface::run_compiler<(), rustc_driver_impl[b803c1eededc3934]::run_compiler::{closure#0}>::{closure#1}, ()>::{closure#0}, ()>::{closure#0}::{closure#0}, ()>::{closure#1}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/thread/mod.rs:557:30
 180:     0x74852dbf8758 - <<std[cd54381a177933ca]::thread::Builder>::spawn_unchecked_<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_with_globals<rustc_interface[bb7f2a6353b38934]::util::run_in_thread_pool_with_globals<rustc_interface[bb7f2a6353b38934]::interface::run_compiler<(), rustc_driver_impl[b803c1eededc3934]::run_compiler::{closure#0}>::{closure#1}, ()>::{closure#0}, ()>::{closure#0}::{closure#0}, ()>::{closure#1} as core[8d9672d5584755be]::ops::function::FnOnce<()>>::call_once::{shim:vtable#0}
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/core/src/ops/function.rs:250:5
 181:     0x74853166b9f4 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::hc89bcfe5b196542d
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/alloc/src/boxed.rs:1966:9
 182:     0x74853166b9f4 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::hdeda1a4e8bbfb163
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/alloc/src/boxed.rs:1966:9
 183:     0x74853166b9f4 - std::sys::pal::unix::thread::Thread::new::thread_start::ha221d69a9add5217
                               at /home/gh-matthiaskrgr/vcs/github/rust_debug_assertions/library/std/src/sys/pal/unix/thread.rs:109:17
 184:     0x748528c9caa4 - start_thread
                               at ./nptl/pthread_create.c:447:8
 185:     0x748528d29c3c - clone3
                               at ./misc/../sysdeps/unix/sysv/linux/x86_64/clone3.S:78:0
 186:                0x0 - <unknown>

error: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md

note: please make sure that you have updated to the latest nightly

note: rustc 1.88.0-dev running on x86_64-unknown-linux-gnu

query stack during panic:
#0 [hir_crate] getting the crate HIR
#1 [opt_hir_owner_nodes] getting HIR owner items in ``
#2 [hir_crate_items] getting HIR crate items
#3 [analysis] running analysis passes on this crate
end of query stack
error: aborting due to 3 previous errors

Some errors have detailed explanations: E0425, E0552, E0658.
For more information about an error, try `rustc --explain E0425`.

```

</p>
</details>

<!--
query stack:
#0 [hir_crate] getting the crate HIR
#1 [opt_hir_owner_nodes] getting HIR owner items in ``
#2 [hir_crate_items] getting HIR crate items
#3 [analysis] running analysis passes on this crate

-->

--- a/compiler/rustc_ast_lowering/src/expr.rs
+++ b/compiler/rustc_ast_lowering/src/expr.rs
@@ -74,14 +74,16 @@ impl<'hir> LoweringContext<'_, 'hir> {
                     // Merge attributes into the inner expression.
                     if !e.attrs.is_empty() {
                         let old_attrs = self.attrs.get(&ex.hir_id.local_id).copied().unwrap_or(&[]);
-                        self.attrs.insert(
-                            ex.hir_id.local_id,
-                            &*self.arena.alloc_from_iter(
-                                self.lower_attrs_vec(&e.attrs, e.span)
-                                    .into_iter()
-                                    .chain(old_attrs.iter().cloned()),
-                            ),
+                        let attrs = &*self.arena.alloc_from_iter(
+                            self.lower_attrs_vec(&e.attrs, e.span)
+                                .into_iter()
+                                .chain(old_attrs.iter().cloned()),
                         );
+                        if attrs.is_empty() {
+                            return ex;
+                        }
+
+                        self.attrs.insert(ex.hir_id.local_id, attrs);
                     }
                     return ex;
                 }

--- a/tests/ui/attributes/invalid-reprs.rs
+++ b/tests/ui/attributes/invalid-reprs.rs
@@ -1,0 +1,6 @@
+fn main() {
+    let y = #[repr(uwu(4))]
+    //~^ ERROR attributes on expressions are experimental
+    //~| ERROR unrecognized representation hint
+    (&id(5)); //~ ERROR: cannot find function `id` in this scope
+}

--- a/tests/ui/attributes/invalid-reprs.stderr
+++ b/tests/ui/attributes/invalid-reprs.stderr
@@ -1,0 +1,33 @@
+error[E0658]: attributes on expressions are experimental
+  --> $DIR/invalid-reprs.rs:2:13
+   |
+LL |     let y = #[repr(uwu(4))]
+   |             ^^^^^^^^^^^^^^^
+   |
+   = note: see issue #15701 <https://github.com/rust-lang/rust/issues/15701> for more information
+   = help: add `#![feature(stmt_expr_attributes)]` to the crate attributes to enable
+   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date
+
+error[E0425]: cannot find function `id` in this scope
+  --> $DIR/invalid-reprs.rs:5:7
+   |
+LL |     (&id(5));
+   |       ^^ not found in this scope
+   |
+help: consider importing this function
+   |
+LL + use std::process::id;
+   |
+
+error[E0552]: unrecognized representation hint
+  --> $DIR/invalid-reprs.rs:2:20
+   |
+LL |     let y = #[repr(uwu(4))]
+   |                    ^^^^^^
+   |
+   = help: valid reprs are `Rust` (default), `C`, `align`, `packed`, `transparent`, `simd`, `i8`, `u8`, `i16`, `u16`, `i32`, `u32`, `i64`, `u64`, `i128`, `u128`, `isize`, `usize`
+
+error: aborting due to 3 previous errors
+
+Some errors have detailed explanations: E0425, E0552, E0658.
+For more information about an error, try `rustc --explain E0425`.

